static/js/map.js

```diff
@@ -320,6 +320,12 @@ function initMap() { // eslint-disable-line no-unused-vars
         }, 500)
     })
 
+    if (showConfig.madmin) {
+        $('#madmin').each(function() {
```

Comment:
Why not put the URL directly on the template?

Reply:
Because the URL is not fixed.

Reply:
What do you mean it is not fixed?
What I see is that upon changing the URL, you still need to restart the app to make the change. So you could put the URL directly in the template, preventing the need to add more javascript to it.

Reply:
Right, it's in the template now.